### PR TITLE
Feature/button tokenization

### PR DESCRIPTION
**Pull request by jsaelhof**
All the theme, theme panel and theme provider work

### CODE REVIEW

File: theme-list/types/themeList.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { ConfigType } from '@stitches/react/types/config';
+
+export type ThemeList = {
+  [key: string]: ConfigType.Theme;
```
Comment (ralvapil) [inline theme-list/types/themeList.ts:4]: would be able to use the ThemeBase type as the restricted key for this type? Not sure if that's overkill.

**Review by ralvapil — APPROVED** [review]
LGTM

File: theme-provider/utils/composeTheme.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { WhiteLabelColors } from '../types/whiteLabelColors';
+import { ThemeBase } from '../../theme';
+import { ConfigType } from '@stitches/react/types/config';
+import { themeList } from '../../theme-list';
+
+export const composeTheme = (themeBase: ThemeBase, colors: WhiteLabelColors = {}): ConfigType.Theme => ({
+  ...themeList[themeBase],
+  colors: {
+    ...themeList[themeBase].colors,
+    ...(colors?.background && { 'surface-primary': colors.background }),
```
[the remark this answers is not in the document]
Comment (jsaelhof) [reply theme-provider/utils/composeTheme.ts:10]: Oh, so this is some placeholder code (line 10/11). What we are expecting is that we will be given a short list of colors...something like the customer's button color, text color etc. Whatever design determines is the list of colors to ask them for. We have to then map those colors to tokens. Something like their "Background" color might be mapped to two or three of our tokens. Same for text color etc. This is the equivalent of a merge but we have to do some mapping inside. I'll add some comments.

File: theme-list/types/themeList.ts (hunk 1 above)
Comment (jsaelhof) [reply theme-list/types/themeList.ts:4]: I tried and it bitched at me.

File: theme-provider/utils/composeTheme.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsaelhof) [reply theme-provider/utils/composeTheme.ts]: adc12c0

**Pull request merged**